ts/Data/DataPool.ts

```diff
@@ -255,6 +255,22 @@ class DataPool implements DataEvent.Emitter {
     }
 
 
+    /**
+     * Tests whether the connector has never been requested.
+     *
+     * @param {string} connectorId
+     * Name of the connector.
```

Comment:
```suggestion
     * ID of the connector.
```

Reply:
Do you call it now connector ID in Dashboards?

Reply:
The argument is called `connectorId` so I thought the description should be the same